tests/bsim/bluetooth/audio/src/bap_broadcast_assistant_test.c

```diff
@@ -424,6 +430,38 @@ static void test_bass_mod_source(void)
 	mod_src_param.pa_interval = g_broadcaster_info.interval;
 	subgroup.bis_sync = BIT(1) | BIT(2); /* Indexes 1 and 2 */
 	subgroup.metadata_len = 0;
+
+	err = bt_bap_broadcast_assistant_mod_src(default_conn, &mod_src_param);
+	if (err != 0) {
+		FAIL("Could not modify source (err %d)\n", err);
+		return;
+	}
+
+	WAIT_FOR_FLAG(flag_cb_called);
+	WAIT_FOR_FLAG(flag_write_complete);
+	printk("Source added, waiting for server to PA sync\n");
+	WAIT_FOR_FLAG(flag_state_synced)
+	printk("Server PA synced\n");
+}
+
+void test_bass_long_write(void)
```

Comment:
```suggestion
void test_bass_mod_source_long_meta(void)
```

Reply:
We don't actually need to do `mod_src` twice - We could just have added the long meta to the existing `test_bass_mod_source`

Reply:
I considered that, and I suppose it would have been good enough since we test the normal read already in the other tests. Leaving it this way so it is obvious that we explicitly test the long read/writes